checker/jdk/index/src/java/lang/String.java

```diff
@@ -162,7 +162,7 @@ public String() {
      * @param  original
      *         A {@code String}
      */
-    public @PolyMinLen @PolySameLen String(@PolyMinLen @PolySameLen String original) {
+    public /*@ PolyMinLen*/ @PolySameLen String(/*@ PolyMinLen*/ @PolySameLen String original) {
```

Comment:
The changeset for this PR will be big until typetools#1247 is merged, because it includes all the changes to the JDK in typetools#1247. I did it programmatically over the whole Index JDK directory.